source/elements/oneMKL/source/domains/dft/descriptor.rst

```diff
@@ -226,37 +271,43 @@ The copy assignment operator results in a deep copy.
    .. rubric:: Input Parameters
 
    other
-      The descriptor to copy or move from.
+      The ``descriptor`` object to copy or move from.
 
 .. container:: section
 
    .. rubric:: Throws
 
-   The assignment opererator shall throw the following exceptions if the associated condition is detected. An implementation may throw additional implementation-specific exception(s) in case of error conditions not covered here:
+   The assignment opererators shall throw the following exceptions if the
+   associated condition is detected. An implementation may throw additional
+   implementation-specific exception(s) in case of error conditions not covered
+   here:
 
    :ref:`oneapi::mkl::host_bad_alloc()<onemkl_exception_host_bad_alloc>`
-      If any memory allocations on host have failed, for instance due to insufficient memory.
-   
+      If any memory allocations on host have failed, for instance due to
+      insufficient memory.
 
 **Descriptor class member table:** :ref:`onemkl_dft_descriptor_member_table`
 
 .. _onemkl_dft_descriptor_set_value:
 
 set_value
-++++++++++
-
-Sets DFT configuration values before :ref:`onemkl_dft_descriptor_commit`.
-
-
-.. rubric:: Description
-
-This function sets one particular configuration parameter with
-the specified configuration value. Each configuration parameter
-is a named constant, and the configuration value must have the
-corresponding type, which can be a named constant or a native
-type. For available configuration parameters and the
-corresponding configuration values, see :ref:`onemkl_dft_enum_config_param`.
-All calls to ``set_param`` must be done before :ref:`onemkl_dft_descriptor_commit`.
++++++++++
+
+The ``set_value`` member function of the ``descriptor`` class sets a
+configuration value corresponding to a (read-write) configuration parameter for
+the DFT(s) that a ``descriptor`` object defines. This function is to be used as
+many times as required for all the necessary configuration parameters to be set
+prior to committing the ``descriptor`` object (by calling its member function
+:ref:`onemkl_dft_descriptor_commit`).
+
+This function requires and expects exactly **two** arguments: it sets the
+configuration value (second argument) corresponding to the configuration
+parameter (first argument) ``param`` of type ``oneapi::mkl::dft::config_param``.
+The type of the configuration value (second argument) to be set depends on the
+value of ``param``: it can be ``oneapi::mkl::dft::domain``,
+``oneapi::mkl::dft::precision``, ``oneapi::mkl::dft::config_value`` or a native
+type like ``std::int64_t`` or ``float`` (more details available
+:ref:`here<onemkl_dft_enum_config_param>`).
```

Comment:
```suggestion
value of ``param``: it can be ``oneapi::mkl::dft::config_value`` or a native
type like ``std::int64_t`` or ``float`` (more details available
:ref:`here<onemkl_dft_enum_config_param>`).
```
I see where domain and precision came from in this list because `FORWARD_DOMAIN` and `PRECISION` are in that `config_param` table. But I think it's confusing to list them here since just the config_value enum or a native type are valid inputs to `set_value`.

Reply:
not to `get_value` though...

Reply:
My bad, I thought this was referring to something else, will change that.

Reply:
Addressed in [66791a78afc1ded03c9083a14391e29b36f9b056](https://github.com/oneapi-src/oneAPI-spec/pull/506/commits/66791a78afc1ded03c9083a14391e29b36f9b056)